docs/core/interactive-learning.rst

```diff
@@ -20,8 +20,7 @@ Some people call this `Software 2.0 <https://medium.com/@karpathy/software-2-0-a
 .. note::
 
     Rasa X provides a UI for interactive learning, and you can use any user conversation
-    as a starting point. See `Copy to Interactive Learning
-    <../../rasa-x/docs/annotate-conversations/>`_ in the Rasa X docs.
+    as a starting point. See `Annotate Conversations <https://rasa.com/docs/rasa-x/annotate-conversations/>`_ in the Rasa X docs.
```

Comment:
rather than posting a link you should be able to do a `:ref:...` so that it changes if we ever change a url

Reply:
I'm not a Sphinx or RST expert but I think that `:ref:` is not an option here because the Rasa X docs are in a different repo than the Rasa docs. Or at least it doesn't work for me locally when I run `make livedocs` in my local `rasa` directory (which makes sense because at that point Sphinx isn't aware of/watching the Rasa X docs at all).

Reply:
:+1: won't work

Reply:
my bad :D